newIDE/app/src/CommandPalette/CommandPalette/AutocompletePicker.js

```diff
@@ -35,14 +53,32 @@ const AutocompletePicker = (
     props.onSelect(item);
   };
 
+  const getItemHint = (item: Item) => {
+    if (item.text) return null;
+    else if (item.name) {
+      const shortcutString = shortcutMap[item.name];
+      if (!shortcutString) return null;
+      const shortcutDisplayName = getShortcutDisplayName(shortcutString);
+      return (
+        <ListItemSecondaryAction>
+          <Chip className={classes.shortcutChip} label={shortcutDisplayName} />
+        </ListItemSecondaryAction>
+      );
+    }
+  };
+
   const getItemText = (item: Item) => {
     if (item.text) return item.text;
-    else if (item.displayText) return props.i18n._(item.displayText);
+    else if (item.name) {
+      const { area, displayText } = commandsList[item.name];
+      const areaText = commandAreas[area];
+      return props.i18n._(areaText) + ': ' + props.i18n._(displayText);
```

Comment:
nit: what do you think of using "〉" instead of ":"? Some languages might not use a colon to separate words :)